docs/mutations.md

```diff
@@ -32,11 +32,11 @@ Do you have a suggestion for a (new) mutator? Feel free to create an [issue](htt
 | `!=` | `==` |
 
 ## Logical Operators (_logical_)
-| Original | Mutated | 
-| ------------- | ------------- | 
-| `&&` | `\|\|` | 
-| `\|\|` | `&&` |
-| `^` | `==` |
+| Original                  | Mutated                   |
```

Comment:
What's the reason for all these changes?

Reply:
Had an outdated fork, should be fixed now 👍